### PR TITLE
  #170947550 add log-out endpoint

### PR DESCRIPTION
**Pull request by bbaime98**
#### What does this PR do?
- add logout functionality
- put all auth tests in one file
- change authController to userController
- change authRoutes to userRoutes
#### Description of Task to be completed?
- add a log-out route
- add a logout function
- add logout tests
#### How should this be manually tested?
-  Clone repository 
- signup first `/api/v1/auth/register`
- login in `/api/v1/auth/login`
- then log out through `/api/v1/auth/logout`
#### What are the relevant pivotal tracker stories?
[#170947550](https://www.pivotaltracker.com/story/show/170947550)
#### Screenshots (if appropriate)
image

#### Questions:

### CODE REVIEW

File: src/services/localesServices/locales/fr.json [hunk 1]
```diff
@@ -1,7 +1,9 @@
...
-    "login": "connecter",
-    "logout": "déconnecter"
-}
+	"Welcome to devRepublic Barefoot Nomad API": "Bienvenue au devRepublic Barefoot Nomad API",
+	"register": "enregistrer",
+	"signup": "enregistrer",
+	"login": "connecter",
+	"logout": "déconnecter",
+	"User is successfully logged out": "l'utilisateur s'est déconnecté avec succès",
+	"Login first or create an account if you do not have one": "Connectez-vous d'abord ou créez un compte si vous n'en avez pas"
```
Comment (Mnickii) [inline src/services/localesServices/locales/fr.json]: one is in english

Comment (bbaime98) [reply src/services/localesServices/locales/fr.json]: I don't get what you mean exactly ?

Comment (bbaime98) [reply src/services/localesServices/locales/fr.json]: I didn't use that last error. I am going to remove it.

**Review by Mnickii — DISMISSED** [review]

File: src/services/localesServices/locales/fr.json (hunk 1 above)
Comment (Mnickii) [reply src/services/localesServices/locales/fr.json]: @bbaime98 you should include the message you get when logged in

Comment (Mnickii) [reply src/services/localesServices/locales/fr.json]: And all possible error and success messages here too for both languages

**Review by Baraka-Mugisha — DISMISSED** [review]

**Review by Ntare22 — CHANGES_REQUESTED** [review]
I can see you renamed the controllers, which seems unnecessary and confusing... Please maintain the names

**Review by Mnickii — DISMISSED** [review]

**Review by Baraka-Mugisha — APPROVED** [review]

**Review by Ntare22 — APPROVED** [review]
LGTM

**Review by Mnickii — APPROVED** [review]

**Pull request merged**